Automattic/newspack-theme
# fix: update custom font selectors in the editor styles

### All Submissions:


### Changes proposed in this Pull Request:

With WordPress 5.7, some of the markup in the editor changed, making it so some custom fonts were no longer applied. 

This PR adds updated selectors, but doesn't override the existing ones, in case the markup of existing blocks wouldn't have changed. 

### How to test the changes in this Pull Request:

1. Start with the default theme, and set up some obvious custom fonts.
2 Set up a post with a variety of blocks that use the header font, or copy [this test content](https://cloudup.com/cmduE5VnrN5) -- it includes a number of the blocks that use header fonts, including header, pullquotes (in some cases), some of the widget blocks, and tables.
3. Publish.
4. Note that some elements in the header - like the title and header blocks aren't picking up the header fonts, when compared to the front end.
5. Apply the PR and confirm that the fonts are now corrected. 
6. Cycle through the different child themes and confirm things are working as expected, compared to the front-end -- the main thing that can change from theme to theme is the dropcap, and pullquote block -- some themes use the body font, and others the header font.

<details>
<summary><strong>Default theme</strong></summary>

<strong>Default theme - before:</strong>

![image](https://user-images.githubusercontent.com/177561/110679234-186af780-818c-11eb-82df-d17410c6b0f1.png)

![image](https://user-images.githubusercontent.com/177561/110679253-1e60d880-818c-11eb-9208-4a406496b362.png)

<strong>Default theme - after</strong>

![image](https://user-images.githubusercontent.com/177561/110678803-8ebb2a00-818b-11eb-8ab2-656ffed34069.png)

![image](https://user-images.githubusercontent.com/177561/110679430-4fd9a400-818c-11eb-85ed-e70db4243b38.png)
</details>


<details>
<summary><strong>Newspack Joseph</strong></summary>

<strong>Newspack Joseph - before:</strong>

![image](https://user-images.githubusercontent.com/177561/110679990-f2922280-818c-11eb-9be2-411e3424c504.png)

![image](https://user-images.githubusercontent.com/177561/110680010-f756d680-818c-11eb-9aa8-96f40f446c15.png)

<strong>Newspack Joseph - after</strong>

![image](https://user-images.githubusercontent.com/177561/110680339-51f03280-818d-11eb-91f9-946b8119dfe7.png)

![image](https://user-images.githubusercontent.com/177561/110680350-56b4e680-818d-11eb-9867-688d4d5c2fec.png)

</details>

<details>
<summary><strong>Newspack Katharine</strong></summary>

<strong>Newspack Katharine - before:</strong>

![image](https://user-images.githubusercontent.com/177561/110680679-c2974f00-818d-11eb-8960-43466f126c07.png)

![image](https://user-images.githubusercontent.com/177561/110680692-c75c0300-818d-11eb-9e74-5442ec9c15c4.png)

<strong>Newspack Katharine - after</strong>

![image](https://user-images.githubusercontent.com/177561/110681086-44877800-818e-11eb-9779-6aaf24362fae.png)

![image](https://user-images.githubusercontent.com/177561/110681106-49e4c280-818e-11eb-908c-5f96020262a3.png)

</details>

<details>
<summary><strong>Newspack Nelson</strong></summary>

_Note: I've switched the custom header font to Georgia for these screenshots, since the original one I was testing with, Raleway, is pretty close to this theme's default._

<strong>Newspack Nelson - before:</strong>

![image](https://user-images.githubusercontent.com/177561/110681954-3d149e80-818f-11eb-9950-a9f613663e5b.png)

![image](https://user-images.githubusercontent.com/177561/110681967-4271e900-818f-11eb-98ab-4502661a948d.png)

<strong>Newspack Nelson - after</strong>

![image](https://user-images.githubusercontent.com/177561/110682374-b613f600-818f-11eb-865e-51c7b0875edd.png)

![image](https://user-images.githubusercontent.com/177561/110682389-bad8aa00-818f-11eb-9621-5d0df1a32aa0.png)

</details>

<details>
<summary><strong>Newspack Sacha</strong></summary>

<strong>Newspack Sacha - before:</strong>

![image](https://user-images.githubusercontent.com/177561/110683731-338c3600-8191-11eb-8592-0c99b9332367.png)

![image](https://user-images.githubusercontent.com/177561/110683752-3850ea00-8191-11eb-9e46-5903ba7720df.png)

<strong>Newspack Sacha - after</strong>

![image](https://user-images.githubusercontent.com/177561/110683934-6d5d3c80-8191-11eb-8578-c04681dc6a34.png)

![image](https://user-images.githubusercontent.com/177561/110683961-71895a00-8191-11eb-999b-33c50c5fe191.png)

</details>

<details>
<summary><strong>Newspack Scott</strong></summary>

<strong>Newspack Scott - before:</strong>

![image](https://user-images.githubusercontent.com/177561/110684318-dc3a9580-8191-11eb-84b8-d0852acb5241.png)

![image](https://user-images.githubusercontent.com/177561/110684347-e65c9400-8191-11eb-87e7-f62452370cfc.png)

<strong>Newspack Scott - after</strong>

![image](https://user-images.githubusercontent.com/177561/110684631-36d3f180-8192-11eb-9b15-cf9b02a68b63.png)

![image](https://user-images.githubusercontent.com/177561/110684650-3cc9d280-8192-11eb-9573-4eefd1039d85.png)

</details>

### Other information:
